src/assets/config.js

```diff
@@ -0,0 +1,10 @@
+
+// See ServicesConfig for format
+window.showcaseConfig = {
+  "backend": {
+    "serverUrl": "http://localhost:4000/graphql",
+    "wsServerUrl": "ws://localhost:4000/graphql"
+  },
+  // "push": {},
+  // "auth": {}
+};
```

Comment:
once this is merged, we should update the datasync deployment template to create a configmap that will have this content and mount it to the pod.